app/services/app/app.ts

```diff
@@ -97,6 +97,25 @@ export class AppService extends StatefulService<IAppState>
 
       this.performanceMonitorService.start();
 
+      // Workaround for the browser source shutdown settings being not applied correctly
+      const activeScene = this.scenesService.activeScene;
+
+      this.scenesService.makeSceneActive(this.scenesService.getScenes()[0].id);
+      this.scenesService.getScenes().forEach((scene,index) => {
+        const previousScene = this.scenesService.activeScene;
+        this.scenesService.makeSceneActive(scene.id);
+        previousScene.items.forEach(item => {
+          let isShowing: Boolean = true;
```

Comment:
You don't need to add a type annotation here since you are immediately assigning it.  Also, the type you would have wanted to use here is `boolean` (lowercase b)

Reply:
Done.